vagrant/README.md

```diff
@@ -0,0 +1,12 @@
+This directory contains a [Vagrantfile](Vagrantfile) for building docker-solr.
+
+Pre-requisites: [VirtualBox](https://www.virtualbox.org/wiki/Downloads) and [Vagrant](https://www.vagrantup.com/downloads.html).
+
+```bash
+vagrant up
+vagrant ssh -c "git clone git@github.com:docker-solr/docker-solr.git && \
```

Comment:
Could not get the git clone to work with ssh, changed to https://github.com/docker-solr/docker-solr.git and it worked

Reply:
I personally prefer the [ssh URLs](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) for repos I have write access to, and I have my git configured such that this works. But, [github recommends https](https://help.github.com/en/github/using-git/which-remote-url-should-i-use), afaict mainly because it works more widely. I'm happy to change it to that.

Reply:
Yes, unless we document prerequisites for the ssh method to work. Are we going to push changes to the repo from within vagrant?

Reply:
I won't be using vagrant myself, but for those who do, presumably yes. If you git push that with a https URL you get prompted for credentials, so that will work too.